usagov/usagov-2021
# USAGOV-1622-life-event-categories: Shows categories for the benefit inder life events.

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-1622

## Description

Adds markup for showing categories column for the benefit finder life events results.

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions


Assumes you have the benefits search page functional on your local instance (see `docs/BenefitsSearch.md` for how to setup)

On the benefits search page, pick at least one category term that refers to a life event and submit the search. The life event result should be at the top with 3 categories

![image](https://github.com/usagov/usagov-2021/assets/493854/2aa895f6-3e90-421d-8193-d9af6fce86df)

### Requires New Config
- [] Yes

### Requires New Content

### Validation Steps
- On the English page, confirm that life event results have 3 categories: "Benefit Finder Tool", "Life Events", and the term name
- On the Spanish page, confirm that life event results have 3 categories in Spanish: "Buscador de beneficios", "Eventos de la vida", and the term name in Spanish

## Security Review




## Reviewer Reminders
- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions
Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
